service/http-api-samples/http-api-owner-sample/src/main/java/org/fido/iot/api/OwnerSviServlet.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Intel Corporation
+// SPDX-License-Identifier: Apache 2.0
+
+package org.fido.iot.api;
+
+import java.io.IOException;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.sql.DataSource;
+import org.fido.iot.protocol.Const;
+import org.fido.iot.storage.OwnerDbManager;
+
+public class OwnerSviServlet extends HttpServlet {
+
+  @Override
+  protected void doPost(HttpServletRequest req, HttpServletResponse resp)
+      throws ServletException, IOException {
```

Comment:
This method doesn't throw any ServletException. It can be removed.

Reply:
Same as above.